scripts/run_lighthouse_tests.py

```diff
@@ -50,11 +61,23 @@ def run_lighthouse_checks():
         python_utils.PRINT(line[:-1])
 
 
+def enable_webpages():
+    """Enables deactivated webpages for testing."""
+    pattern = 'COMMUNITY_DASHBOARD_ENABLED = .*'
+    replace = 'COMMUNITY_DASHBOARD_ENABLED = True'
+    common.inplace_replace_file(FECONF_FILE_PATH, pattern, replace)
+
+    pattern = '"ENABLE_ACCOUNT_DELETION": .*'
+    replace = '"ENABLE_ACCOUNT_DELETION": true,'
+    common.inplace_replace_file(CONSTANTS_FILE_PATH, pattern, replace)
+
+
 def main():
     """Runs lighthouse checks and deletes reports."""
-    setup_and_install_dependencies()
+    # setup_and_install_dependencies()
+    enable_webpages()
     run_lighthouse_checks()
-    delete_reports()
+    clean_up()
```

Comment:
Please run register this as `atexit` function, you can look into `run_e2e_tests.py` line 443, (`atexit.register(cleanup)`).

Reply:
Don't forget to register it before the `run_lighthouse_checks()`.

Reply:
Done